### ISSUE TITLE
Handling of SDPs with number of ports specified on m line

### ISSUE THREAD

**Issue by denislemire** [sipwise/rtpengine#610]
We're seeing an issue where when a client specifies the number of ports as in RFC-4566 section 5.14 the call fails, example SDP:

```
v=0
o=892031 0 0 IN IP4 10.0.1.11
s=Samsung IP PBX
c=IN IP4 10.0.1.13
t=0 0
m=audio 40314**/2** RTP/AVP 0 101
c=IN IP4 10.0.1.13
a=rtpmap:0 PCMU/8000
a=rtpmap:101 telephone-event/8000
a=sendrecv
```

In the RTPENGINE logs for this scenario we'll consistently get "Failed to handle consecutive ports" regardless of port utilization by the RTPENGINE instance.

Are SDPs of this form supported by RTPENGINE?

**Comment by rfuchs** [comment]
Yes that's supposed to work, even though this is the first real life use case of it that I've seen. Will look into it.
